addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -53,7 +53,7 @@ def _l10n_it_edi_generate_electronic_invoice_filename(self, invoice):
 
         return '%(country_code)s%(codice)s_%(progressive_number)s.xml' % {
             'country_code': invoice.company_id.country_id.code,
-            'codice': invoice.company_id.l10n_it_codice_fiscale.replace(' ', ''),
+            'codice': (self.env['res.partner']._l10n_it_normalize_codice_fiscale(invoice.company_id.l10n_it_codice_fiscale)).replace(' ', ''),
```

Comment:
@dbkosky Could we remove the replace?  (it makes things more clear at least, it just spreads uncertainty for the moment)